makeybutton/button.go

```diff
@@ -61,7 +67,7 @@ func (b *Button) Get() ButtonEvent {
 	b.readings.Put(pressed)
 
 	switch {
-	case pressed && avg > 0:
+	case pressed && avg > -1*bufferSize-2:
```

Comment:
bufferSize is 6, avg could take a value between -6 and +6 
-1*bufferSize-2 is -8
avg > -1*bufferSize-2 willl always be true¿?

Reply:
Oops! Corrected.